students/minchan/westagram/westagram/urls.py

```diff
@@ -1,4 +1,6 @@
 from django.urls import path
+from users.views import user_register
 
 urlpatterns = [
-]
+    path('minchan/user_register', user_register.as_view(), name='user_register'),
```

Comment:
URI에서 동사는 http Method(POST, GET, DELETE ..) 있기 때문에 user만으로 충분합니다.

```suggestion
    path('user/', user_register.as_view(), name='user_register'),
```

Reply:
다른 분들 프로젝트도 많이 있는데, 제 이름을 넣어서 구분하지 않아도 괜찮은가요?

Reply:
네 괜찮습니다! 미션 이지만 실제로 배포한다고 생각해주세요~